protected/views/adminFile/view.php

```diff
@@ -8,17 +8,32 @@
 
 <?php
 $sample_id = FileSample::model()->find('file_id=:file_id', array(':file_id'=>$model->id));
- if(isset($sample_id))
+$attributes = FileAttributes::model()->findAll('file_id=:file_id', array(':file_id'=>$model->id));
+
+if(isset($sample_id))
  {
  $sample_name= Sample::model()->find('id=:id',array(':id'=>$sample_id->sample_id));
  }
- 
+
  $name="Not Set";
  
  if(isset($sample_id)&&isset($sample_name))
  {
      $name=$sample_name->name;
  }
+
+$attribute_id = "";
+
+ if(!empty($attributes))
+ {
+     foreach ($attributes as $attribute)
+     {
+         $attribute_id .= $attribute->id . " ";
+     }
+ }else{
+     $attribute_id = "Not Set";
```

Comment:
If a `File` object does not have a `Sample` or `Attribute` associated with it then I would not display these rows at all in the admin file view page.

Reply:
Updated the table to show `File Attribute Value` row only if it is not empty, otherwise, the row will not be displayed.